src/any_agent/frameworks/any_agent.py

```diff
@@ -149,6 +158,8 @@ def _setup_tracing(self) -> None:
             return
         self._exporter = AnyAgentExporter(self.framework, self._tracing_config)
         self._tracer_provider.add_span_processor(SimpleSpanProcessor(self._exporter))
+        for exporter in self._tracing_config.additional_exporters:
+            self._tracer_provider.add_span_processor(SimpleSpanProcessor(exporter))
```

Comment:
Is this code related to the integration test? Seems like it's a change to the API (which if so should be it's own ticket and PR)

Reply:
Hum. It is in fact a non-breaking change in the api that helps checking the agent results when working as a served entity. But I think there are other comments re: this so I'll explain in depth there.